src/ReactiveDomain.Users/Sevices/UserSvc.cs

```diff
@@ -25,18 +25,25 @@ public class UserSvc :
         /// </summary>
         /// <param name="repo">The repository for interacting with the EventStore.</param>
         /// <param name="bus">The dispatcher.</param>
-        public UserSvc(IRepository repo)
+        public UserSvc(IRepository repo, IDispatcher bus)
         {
-            _repo = new CorrelatedStreamStoreRepository(repo);                 
-        }      
+            _repo = new CorrelatedStreamStoreRepository(repo);
+            bus.Subscribe<UserMsgs.CreateUser>(this);
+            bus.Subscribe<UserMsgs.Deactivate>(this);
+            bus.Subscribe<UserMsgs.Activate>(this);
+            bus.Subscribe<UserMsgs.UpdateUserDetails>(this);
+            bus.Subscribe<UserMsgs.MapToAuthDomain>(this);
+            bus.Subscribe<UserMsgs.AddClientScope>(this);
+            bus.Subscribe<UserMsgs.RemoveClientScope>(this);
```

Comment:
These subscriptions all leak since they're never disposed. Let's make this class a `TransientSubscriber` so this will get cleaned up automatically.